test/distributed/fsdp/test_fsdp_mixed_precision.py

```diff
@@ -872,21 +900,27 @@ def test_full_precision_in_eval_buffers(self):
 
     @skip_if_lt_x_gpu(2)
     def test_full_precision_in_eval_comm(self):
-        for use_composable, cast_forward_inputs in itertools.product(
-            [True, False], [True, False]
+        for use_composable, cast_forward_inputs, full_prec_in_eval in itertools.product(
+            [True, False], [True, False], [True, False]
         ):
+            os.environ["FSDP_FULL_PREC_IN_EVAL"] = "1" if full_prec_in_eval else "0"
             mp_config = MixedPrecision(
                 param_dtype=torch.float32,
                 reduce_dtype=torch.float16,
                 buffer_dtype=torch.float32,
                 cast_forward_inputs=cast_forward_inputs,
+                # cast reduction for batchnorm also just in this test, to make
+                # validation easier.
+                _module_classes_to_ignore=[],
             )
             model = TransformerWithSharedParams.init(
                 self.process_group,
                 FSDPInitMode.NO_FSDP if use_composable else FSDPInitMode.RECURSIVE,
                 CUDAInitMode.CUDA_BEFORE,
                 {"mixed_precision": mp_config},
             )
+            for fsdp_module in FSDP.fsdp_modules(model):
+                mp = fsdp_module.mixed_precision
```

Comment:
Where is `mp` used? Why do we take the last one of the FSDP modules?

Reply:
Why is this resolved? :)

Reply:
oops, I deleted this locally but somehow didn't push that change.